sts_synindex_external.R

```diff
@@ -86,22 +88,26 @@ unlink(PARQUET_FILTERED_LOCATION, recursive = T, force = T)
 # De-identify parquet datasets --------------------------------------------
```

Comment:
Every time you commit code, you can do some light "quality of life" maintenance.  For example, you can take this opportunity to shift all R functions to the very top so users can separate out the functions from the steps of code executed